api/preprints/serializers.py

```diff
@@ -464,7 +464,8 @@ def require_admin_permission():
             description = validated_data['description']
             self.set_field(preprint.set_description, description, auth)
 
-        preprint.article_doi = validated_data.get('article_doi')
+        if 'article_doi' in validated_data:
```

Comment:
This was the bad line that was wiping the `article_doi` when it PATCHed with an empty dict.